app/assets/stylesheets/components/_buttons.scss

```diff
@@ -1,5 +1,14 @@
 button,
+input[type="submit"],
 .btn {
   background-color: $color-buttons-main;
   border: 0;
+  color: $white;
+  font-weight: normal;
+  margin-bottom: 2em;
+  padding: 8px 24px 10px;
+
+  &:hover{
+    background-color: $orange-dark;
```

Comment:
I like the uses of color variables throughout this file. But something like this, instead of being a variation on a root color, used once, might be better off as a variation on a _local_ color. The navbar has a `$color-buttons-main` which is currently set to `$orange`, so it might be more informative to do the darkening here instead.

```scss
 .btn {
   background-color: $color-buttons-main;

  &:hover {
    background-color: darken($color-buttons-main, 8%);
}